src/Auth/AccessTokenClient.php

```diff
@@ -25,7 +25,7 @@ public function __construct(IHttpClient $client)
 		$this->client = $client;
 	}
 
-	public function getAccessToken(Config $config): AccessToken
+	final public function getAccessToken(Config $config): AccessToken
```

Comment:
Proc je to final?

Reply:
Protože mi to přijde bezpečnější, předtím byla metoda přetížená a musela se duplikovat logika viz `AccessTokenClient::getAccessToken()`. Co bude potřeba vyřešit se vyřeší přetížením `AccessTokenCacheProvider::generateAccessToken`. Pokud bude potřeba jiná implementace, tak máme rozhraní.